scripts/src/models/translations.ts

```diff
@@ -18,9 +20,15 @@ export class OfferTranslation extends BaseEntity {
 
 	public static async getTranslations(criteria: GetTranslationsCriteria = {}): Promise<OfferTranslation[]> {
 		//  todo add cache
+		const cacheKey = JSON.stringify(criteria);
```

Comment:
stringify will give you different dumps depending on the structure in memory, no?

Reply:
mmm there is a chance it will change the order, ok I'll change it just in case.